app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -4,21 +4,20 @@
   contact_us = Rails.configuration.x.organisation.contact_us_url || contact_us_url
   email_subject = _('Query or feedback related to %{tool_name}') %{ :tool_name => tool_name }
 %>
-<% I18n.with_locale I18n.default_locale do %>

```

Comment:
This PR just removes the use of the default locale

Reply:
looks good!